staging/src/k8s.io/component-helpers/apimachinery/lease/controller_test.go

```diff
@@ -478,7 +479,7 @@ func setNodeOwnerFunc(c clientset.Interface, nodeName string) func(lease *coordi
 					},
 				}
 			} else {
-				klog.Errorf("failed to get node %q when trying to set owner ref to the node lease: %v", nodeName, err)
+				klog.Background().Error(err, "failed to get node when trying to set owner ref to the node lease", "node", nodeName)
```

Comment:
Using `klog.Background().Error` is the same as calling `klog.ErrorS`: it's structured logging, but not contextual logging.

It's quite likely that APIs will have to be changed to support passing a logger from the caller. Let's first decide about promoting contextual logging to beta in 1.29 (will file a KEP update soon).